elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -0,0 +1,142 @@
+@import "../../pfe-sass/pfe-sass";
+$LOCAL: modal;
+
+$pfe-modal--breakpoint--medium: 640px;
+
+:host {
+	--pfe-modal--MaxWidth: 70vw;
+	--pfe-modal--MinWidth: 50vw;
+	--pfe-modal--MaxWidth--mobile: 94vw;
+	--pfe-modal--MaxHeight: 90vh;
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) 0 calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
+	--pfe-modal--Padding--mobile: 0 0 0 calc(#{pfe-var(container-padding)} * 2);
+	--pfe-modal__overlay--BackgroundColor: #{pfe-color(overlay)};
+	--pfe-modal--CloseButtonSize: calc(#{pfe-var(ui--element--size)} - 4px);
+	display: block;
+	position: relative;
+}
+
+:host(:not([has_header])) {
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 4) calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
+}
+
+.pfe-modal {
+	&__outer {
+		display: flex;
+		position: fixed;
+		height: 100%;
+		width: 100%;
+		top: 0;
+		left: 0;
+		flex-direction: column;
+		align-items: center;
+		justify-content: center;
+		z-index: var(--pfe-z--modal, 2000);
```

Comment:
Let's add the z-index map to this branch too.  This line would become `z-index: #{pfe-var($cssvar: modal, $map: $z-index)};` and in pfe-sass in the `_maps` file, can you add:
```
$z-index: (
  modal: 99,
  navigation: 98,
  overlay: 97,
  content: 0
);
```

Reply:
I like the idea of all z-index's starting below 100 if we can and building our layers from there.  If needed, someone could customize their z-indexes in their theme attributes.

Reply:
Yes! I was going to ask about using the z-index map from the nav. I wonder if we should set it higher than 99 since the sticky nav on rh.com is at 1000.

Reply:
@castastrophe maybe it makes more sense to remove the z-index map and add it back later (when the nav is actually at z-index: 98)?

Reply:
I have removed this and left a comment that we should revisit this once the nav is merged in and in production.

Reply:
We can set it higher in our dotCom theme but I think for the general PFE library, starting at 100 should be plenty of levels.

Reply:
Yours will be merged in first and since you need the z-index for it to work, go ahead and copy in the z-index map to your project.

Reply:
@castastrophe added this back.